core/domain/stats_domain.py

```diff
@@ -117,30 +117,28 @@ class StateStats(object):
     """Domain object representing analytics data for an exploration's state."""
 
     def __init__(
-            self, total_answers_count, useful_feedback_count,
-            learners_answered_correctly, total_hit_count, first_hit_count,
-            total_solutions_triggered_count):
+            self, total_answers_count, useful_feedback_count, total_hit_count,
+            first_hit_count, total_solutions_triggered_count, total_finishes):
         """Constructs a StateStats domain object.
 
         Args:
             total_answers_count: int. Total number of answers submitted to this
                 state.
             useful_feedback_count: int. Total number of answers that received
                 useful feedback.
-            learners_answered_correctly: int. Number of learners who submitted
-                correct answers to the state.
             total_hit_count: int. Total number of times the state was entered.
             first_hit_count: int. Number of times the state was entered for the
                 first time.
             total_solutions_triggered_count: int. Number of times the solution
                 button was triggered to answer a state.
+            total_finishes: int. Number of times the state was finished.
```

Comment:
I think total_completions is more idiomatic than total_finishes (in English). Might want to change names accordingly elsewhere.

Reply:
Done.